src/components/AppLayout/NavHeader.tsx

```diff
@@ -86,6 +88,19 @@ const NavHeader = ({ className, sidedrawer, setSidedrawer, displayName, isVerifi
 	const dispatch = useDispatch();
 	const { resolvedTheme: theme } = useTheme();
 
+	const getAssetUsdPrice = async () => {
```

Comment:
this function doesn't make sense in Navheader.
It's also missing a netowkr check.

check something like : `networkConstants[Network].supportedAssets.includes(NetworkAssets.DED.name)`